package.json

```diff
@@ -51,5 +51,8 @@
     "underscore": "~1.8.3",
     "webpack": "~3.5.6"
   },
+  "optionalDependencies": {
+    "babel-core": "^6"
```

Comment:
This seems like a scary thing to add. Won't this mean that every person using CoffeeScript will install Babel automatically, even if they aren't using it? It also limits people to only using Babel 6.x for the duration of CoffeeScript 2.x's lifetime, but we're getting pretty close to releasing 7.x and don't plan to really support 6.x much once that has happened.

Reply:
`npm` automatically installs optional dependencies unless you explicitly install with `--no-optional`, it just doesn't error if they fail to install for some reason, so this is essentially the same as having a full dependency on `babel-core` since I can't think of a time `babel-core` would fail to install